src/validator-map.js

```diff
@@ -60,7 +60,7 @@ VALIDATOR_MAP[DATA_TYPES.TIME] = Utils.buildRegexCheck('isTime');
 // 1, 2, 3, +40, 15,121
 const intRegexCheck = Utils.buildRegexCheck('isInt');
 function isInt(value) {
-  if (intRegexCheck(value)) {
+  if (intRegexCheck(value) || value == '0') {
```

Comment:
```suggestion
  if (intRegexCheck(value) || value === '0') {
```
Do we know that `value` is a string? If not, should we also check for `value === 0`?